src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/model/OffenderManagementUnit.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.prisonregister.model
+
+import org.hibernate.annotations.GenericGenerator
+import org.hibernate.annotations.Parameter
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.OneToOne
+import javax.persistence.PrimaryKeyJoinColumn
+
+@Entity
```

Comment:
This is more like a component than an entity - It doesn't have its own identity and it is owned by and its life-cycle is contained by a Prison. 
Making it an Entity means that there's a Repository available.